Printout alerts

lostie opened the pull request:
[Trello #30](https://trello.com/c/YbFEfMIj/30-as-someone-printing-out-a-completed-per-i-want-to-see-all-the-detainee-s-risk-markers)

- Change print layout to contain the sections related with the move
alerts
- Adds a presenter for the move alerts and a presenter for a specific
alert.

<img width="652" alt="screen shot 2017-03-13 at 16 24 21" src="https://cloud.githubusercontent.com/assets/1006365/23864202/af1b38bc-0809-11e7-8875-dc324981505b.png">

Commit:
Add alert components to the printout.

- Change print layout to contain the sections related with the move
alerts
- Adds a presenter for the move alerts and a presenter for a specific
alert.

Commit:
Configure alert presentation for PDF conversion

This is just a fix to get the alerts to display. We could do with
revisiting this to make it consistent with the other presenters - others
not having a view_context. The CSS looks like it can be DRY’d up a bit.

Commit:
Fix alert on styles for pdf

Still needs a bit of work to get the padding correct. At present the
turned on alerts are slightly bigger than their turned off counterparts.

Commit:
Enhance alert boxes on PDF document

Stop alerts from being duplicated across pages

Update the padding when the alert is turned on. Add rounded corners to
match the design.

Commit:
Remove overlapping table header on PDF

There is a problem with the table header overlapping text after the
first page of details, this commit removes that for now. We will have
to come up with a way to add the detainee identifier to pages eventually.

This commit updates the specs. We need to come up with a better way of
testing this. The more content we have the harder it is. Right now if
we make a change to the PDF updating the spec becomes a time sink.

dan-palmer commented:
<img width="497" alt="screen shot 2017-03-13 at 15 21 14" src="https://cloud.githubusercontent.com/assets/1006365/23864099/6d6baa82-0809-11e7-94b5-7ac4b9cdb6dc.png">

Commit:
Configure wicked pdf to use wkhtmltopdf-binary

Ensure the app uses the correct version of wkhtmltopdf. At present we
expect version 0.12.3 to be used. CircleCI uses 0.9 if it is allowed
to just look up the PATH.

Commit:
Check full text instead of individual lines on PDF

Commit:
Dry status check in print alerts presenter

Commit:
Hide various alert helping methods from public api
